src/main/resources/application.properties

```diff
@@ -52,3 +52,5 @@ spring.data.redis.port=\${REDIS_PORT}
 spring.data.redis.ssl=\${REDIS_SSL_ENABLED}
 wallet.session.ttl=\${WALLET_SESSION_TTL}
 spring.data.mongodb.uuid-representation=standard
+# Migration
+wallet.migration.cardPaymentMethodId=\${WALLET_MIGRATION_CARD_PAYMENT_METHOD_ID}
```

Comment:
in dev (helm value) you can use `148ff003-46a6-4790-9376-b0e057352e45` @petretiandrea